tools/serverpod_cli/test/analyzer/models/stateful_analyzer/model_validation/field/field_datatype_test.dart

```diff
@@ -12,13 +12,15 @@ void main() {
       'bool',
       'int',
       'double',
-      'Uuid',
```

Comment:
Question: Was this previously a validation error?

Reply:
No but it should have been.